content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,207 @@
+============
+Event tracks
+============
+
+Odoo *Events* provides users with the ability to create, schedule, and manage talks, lectures,
+demonstrations, presentations, etc., which are otherwise known as *Tracks*.
```

Comment:
```suggestion
demonstrations, presentations, etc.. These are otherwise known as *Tracks*.
```

Reply:
I'm personally not the "biggest fan" of the double-period to end a sentence. But I fully understand where you're coming from with this suggestion. However, since both are still grammatically correct, I think I'm going to opt for sticking with its current formatting. But, that said, if any other reviewer sees this as an issue, I'll readily make the change (and say Tim saw it first! 😆 ) Thanks!